hypervisor/src/lib.rs

```diff
@@ -150,7 +150,7 @@ pub enum IoEventAddress {
 }
 
 #[derive(Clone, serde::Serialize, serde::Deserialize)]
-#[allow(clippy::large_enum_variant)]
+//#[allow(clippy::large_enum_variant)]
```

Comment:
Why is this just commented out? Also your use of the term "clippy" is bad - these are "clippy directives" or "clippy suppressions.

Reply:
Sorry, I was testing and pushed.

Reply:
I missed mshv+kmv combination of test. I will drop this commit.